Sprint-3/revise/implement/get-ordinal-number.js

```diff
@@ -0,0 +1,29 @@
+
+function getOrdinalNumber(num) {
+    // If the number is 0 or negative, return it as-is (no ordinal suffix).
+    if (num <= 0) return `${num}`;
+
+    // Define the suffixes for ordinal numbers.
+    const suffixes = ["th", "st", "nd", "rd"];
+
+    // Extract the last two digits to handle special cases (e.g., 11th, 12th, 13th).
+    const value = num % 100;
+
+    // Return the number with the appropriate suffix.
+    // Use modulus to determine the correct suffix or fall back to "th".
+    return `${num}${suffixes[(value - 20) % 10] || suffixes[value] || suffixes[0]}`;
```

Comment:
Can you elaborate how this expression yield the suffix value when `value` is 11?
`suffixes[(value - 20) % 10] || suffixes[value] || suffixes[0]`